SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -185,6 +188,8 @@
 
   private long startNewReconcileEverySeconds = TimeUnit.MINUTES.toSeconds(10);
 
+  private TimeZone timeZone = TimeZone.getTimeZone("UTC");
```

Comment:
Yes, it is still able to start up.
I renamed the field to mailerTimeZone, and putting a top level line 'mailerTimeZone: UTC' into the yaml doesn't break the build.